tests/integration2/README.md

```diff
@@ -0,0 +1,82 @@
+# Istio Integration Tests
+
+This folder contains Istio integration tests that use the test framework checked in at 
+[istio.io/istio/pkg/test/framework](https://github.com/istio/istio/tree/master/pkg/test/framework).
+
+### Basics
+
+The goal of the framework is to make it as easy to author and run tests as possible. In it is most simplest
+case, just typing ```go test ./...``` should be sufficient to run tests.
+
+The test framework is designed to work with standard go tooling and allows developers
+to write environment-agnostics tests in a high-level fashion. The quickest way to get started with authoring
+new tests is to checkout the code in 
+[examples](https://github.com/istio/istio/tree/master/tests/integration2/examples) folder.
+
+The test framework has various flags that can be set to change its behavior.
+
+ 
+### Environments
+
+The test framework currently support two environments:
+ 
+  * **Local**: The test binaries run either in-memory, or locally as processes.
+  * **Kubernetes**: The test binaries run in a Kubernetes cluster, but the test logic runs in the test binary.
+  
+When running tests, only one environment can be specified:
+
+```console
+$ go test ./... -istio.test.env local
+$ go test ./... -istio.test.env kubernetes
+```
+
+By default, the tests will be executed in the
+specified environment. However, test authors can specifically indicate a particular environment as
+a requirement for their tests. If the indicated environment does not match the environment that is required
+by the test, then the test will be skipped.
+
+```go
+// Will be skipped in kubernetes environment.
+func TestLocalOnly(t *testing.T) {
+	framework.Requires(t, dependency.Local)
+	// ...
+}
+```
+
+When an environment is not specified, the tests will run against the local environment by default.
+
+#### Kubernetes Environment
+
+When running the tests against the Kubernetes environment, you will need to explicitly specify a kube config
+file for the cluster you need to use. **Be aware that the target cluster will be changed destructively**.
```

Comment:
destructively sounds pretty ominous. Maybe expand on that a little?

Reply:
Ominous would accomplished the intended goal. :)

Explained a bit further.